.env.template

```diff
@@ -9,4 +9,9 @@ AZURE_WEBHOOK=Your.Azure.webhook
 
 MONGO_CONNECTION_STRING=Your.mongo.connection.string
 MONGO_DATABASE_NAME=your.database.name
-MONGO_COLLECTION_NAME=your.collection.name
\ No newline at end of file
+MONGO_COLLECTION_NAME=your.collection.name
+
+GITHUB_OAUTH_CLIENT_ID=ID HERE
```

Comment:
Thanks for remembering the update this, makes my (and other new dev's) lives much easier if they come to this project later!